src/main/java/org/javacord/bot/Main.java

```diff
@@ -17,19 +16,42 @@
 import org.javacord.bot.commands.SetupCommand;
 import org.javacord.bot.commands.WikiCommand;
 
-public class Main {
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
-    private static Logger logger = LogManager.getLogger(Main.class);
+public class Main {
 
     /**
-     * The entrance point of the bot.
+     * The entry point of the bot.
      *
-     * @param args The first argument should be the bot's token. Every other argument gets ignored.
+     * @param args The bot requires exactly one argument, either a file with the token as content or the token directly.
+     *             If the argument is a relative file path, it is relative to the working directory.
      */
```

Comment:
Checkstyle needs `@throws` declaration here.
```suggestion
     * @throws IOException If an error occured while reading the token from a file.
     */
```